docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -88,7 +91,9 @@ However, `Type` may be explicitly set to `oneshot` for `.container` and `.kube`
 to run once `podman` exits.
 
 When setting `Type=oneshot`, it is recommended to also set `RemainAfterExit=yes` to prevent the service state
-from becoming `inactive (dead)`
+from becoming `inactive (dead)`. However, when activating a service via a timer unit, having `RemainAfterExit=yes`
+leaves the job in a "started" state which prevents subsequent activations by the timer. For more information, see the
```

Comment:
leaves the job in **the** "started" state

Reply:
For more information see the systemd.service(5) man page. or
For more information see systemd.service(5)